resources/keeper.yaml

```diff
@@ -192,10 +198,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: status.podIP
-            - name: NODE_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: spec.nodeName
+            - name: ETCD_HOSTNAME
+              value: "kubernetes.default.svc.cluster.local"
```

Comment:
why are you using `kubernetes.default.svc.cluster.local` here instead of just `127.0.0.1` ?

Reply:
I just forgot to delete it. etcd endpoint address is configurable through STORE_ENDPOINTS env variable, or, by default, it is `127.0.0.1:2379`.